src/optionsManager.js

```diff
@@ -35,6 +34,14 @@ export default {
     saveToLocalStorage(`consoleOptions.${vmId}`, JSON.stringify(options))
   },
 
+  clearConsoleOptions () {
```

Comment:
How about e.g. `clearAutoConnect`. The name seems a bit misleading for me since many console options in local storage may survive a run of this method.